wondrous-app/components/organization/pods/PodItem/index.tsx

```diff
@@ -0,0 +1,89 @@
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import RolePill from 'components/Common/RolePill';
+import PodIcon from 'components/Icons/podIcon';
+import MemberIcon from 'components/Icons/Sidebar/people.svg';
+import { LockedIconOutline, LockIconOutline } from 'components/Icons/userpass';
+import palette from 'theme/palette';
+import { useOrgBoard, useTokenGating } from 'utils/hooks';
+import { PodItemContainer, PodItemStatsContainer } from './styles';
+
+const PodItem = (props) => {
+  const { podData } = props;
+
+  const { userPermissionsContext } = useOrgBoard() || {};
+  const [tokenGatingConditions] = useTokenGating(podData?.orgId);
+
+  const podId = podData?.id;
+  const bgColor = podData?.color || palette.black85;
+  const podName = podData?.name;
+  const podDescription = podData?.description;
+  const contributorCount = podData?.contributorCount || 0;
+  const isPodTokenGated =
```

Comment:
this is my bad i should've noted, let's just remove the token gating stuff, it doesn't make a lot of sense on pod level